.github/workflows/multi-node.yml

```diff
@@ -19,28 +20,35 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
+
       - name: Install Kubectl
         run: |
           sudo snap install kubectl --classic
       - uses: google-github-actions/setup-gcloud@v0.2
         with:
           service_account_key: ${{ secrets.GKE_SA_KEY }}
           project_id: ${{ secrets.GKE_PROJECT }}
+
       - name: Create the cluster
         run: |-
           gcloud config set compute/region us-central1
           gcloud config set compute/zone us-central1-c
           ./kubernetes/create-cluster-gke.sh "akka-multi-node-${GITHUB_RUN_ID}"
+
       - name: Setup Pods
         run: |
-          # Stress tests are using 13 nodes
-          ./kubernetes/setup.sh 15 multi-node-test.hosts
+          # Stress tests are using 13 nodes.
+          # on port 5000 for TCP
+          ./kubernetes/setup.sh 15 5000 multi-node-test.hosts
```

Comment:
We should pass only one port and decide here which one we will use. 
For TCP, 5000. For UDP, 6000.

Reply:
are you sure that would work? when we run with ordinary TCP, some tests are still configured to run with Aeron UDP

Reply:
Hmm, that's then probably the reason why we had this in a few tests.

https://github.com/akka/akka/pull/30706/files#diff-0435f72b22aba40df8eb7c4d7433d99100610a622addbc36f5ff387838c757c2L82

I will dig it further.